tgui/packages/tgui/interfaces/Adminhelp.tsx

```diff
@@ -71,8 +71,8 @@ export const Adminhelp = (props) => {
                       onChange={(value) => {
                         if (value === confirmationText) {
                           setRequestForAdmin(true);
+                          setCurrentlyInputting(false);
```

Comment:
how does this work? should this maybe be put on onBlur instead?

Reply:
I believe? This is what it was originally going for, this change makes the input stay until you input the phrase, after which it swaps back to a (now active) button

Reply:
onChange is each key, there's an `onBlur` event on input too